src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/LegacyRazorCompletionEndpointTest.cs

```diff
@@ -609,7 +609,7 @@ public async Task Handle_ProvidesTagHelperAttributeItems()
 
     private static RazorCodeDocument CreateCodeDocument(string text)
     {
-        var codeDocument = TestRazorCodeDocument.CreateEmpty();
+        var codeDocument = TestRazorCodeDocument.Create(text);
```

Comment:
Same comment here as for Code Action tests. If we don't specify text, we end up with empty Source document in created  code document and any passed in position (other than (0, 0)) fails to resolve. This was passing only because were weren't checking line positions.